vectors/cryptography_vectors/x509/custom/rsae_cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Any new vectors need to be documented in `test-vectors.md`.

Reply:
Oh nice, thanks to this documentation I could reuse another RSA key for my test. Only needed to add a single file, which is now documented.